cookbooks/Gradio/huggingface.aiconfig.json

```diff
@@ -221,7 +230,14 @@
         },
         "parameters": {}
       },
-      "outputs": []
+      "outputs": [
+        {
+          "output_type": "execute_result",
+          "execution_count": 0,
+          "data": " Hi.",
+          "metadata": {}
+        }
+      ]
```

Comment:
noticed the asr prompt didn't have an output persisted but all the other ones did